docs/design/autoware-interfaces/ad-api/list/api/pose/index.md

```diff
@@ -0,0 +1,18 @@
+# Pose API
+
+- {{ link_ad_api('/api/pose/state') }}
+- {{ link_ad_api('/api/pose/initialize') }}
+
+## Description
+
+This API manages the initialization of the vehicle pose. Autoware requires a global pose as the initial guess for localization.
+
+## States
+
+![pose-reset-state](./state.drawio.svg)
+
+| State        | Description                                                            |
+| ------------ | ---------------------------------------------------------------------- |
+| UNAVAILABLE  | The vehicle pose is not available. Waiting for initialization request. |
+| INITIALIZING | The vehicle pose is not available. Initialization is in progress.      |
+| AVAILABLE    | The vehicle pose is available. Initialization can be requested again.  |
```

Comment:
I think it is okay with `AVAILABLE`, because the purpose of this API is to manage the status of the initialization.

Reply:
It's okay, but then I think it's good to rename this API so that we can understand it's only for initialization.

Reply:
yes. I agree with @kenji-miyake san.

Reply:
@kenji-miyake @YamatoAndo 
Initially this was the name `pose_initialization`. However, the name `initial pose` is misleading as Autoware initialization. So I removed initialization from this.

Reply:
> the name initial pose is misleading as Autoware initialization

@isamu-takagi Why did you feel so?
Also, do you mean you want to keep the current definition?

Reply:
How about `localization`?
- `/api/localization/initialize`
- `/api/localization/state  (UNAVAILABLE, INITIALIZING, AVAILABLE)`

Reply:
> Why did you feel so?

@kenji-miyake 
Because this discussion https://github.com/orgs/autowarefoundation/discussions/367

Reply:
`localization` seems good.

> /api/localization/state (UNAVAILABLE, INITIALIZING, AVAILABLE)

But I personally feel this one is better.

`/api/localization/initialization_state (UNINITIALIZED, INITIALIZING, INITIALIZED)`

Reply:
I am concerned about [this discussion](https://github.com/autowarefoundation/autoware_core_universe_prototype/discussions/367) for the word `initialization`.
I don't feel so (maybe because I'm a developer).

How do you feel? @mitsudome-r @kenji-miyake @yukkysaito @YamatoAndo 

```
/api/localization/state
/api/localization/initialization_state
/api/pose_reset/state
/api/pose/initialization/state
```

Reply:
Sorry, I cannot understand about `UNINITIALIZED, INITIALIZING, INITIALIZED` by reading the readme.
pls explain or write to readme more detail and examples of when to transition to that state.

Reply:
```
UNINITIALIZED: The initial state or after initialization has failed.
INITIALIZING: During running the initialization API.
INITIALIZED: After initialization has succeeded.
```

Since `AVAILABLE` is unclear as explained in this thread, I think it's good to focus on initialization.

Reply:
@kenji-miyake Thank you :+1: 
@isamu-takagi Can you explain about `UNAVAILABLE, INITIALIZING, AVAILABLE` ?

Reply:
> I am concerned about [this discussion](https://github.com/autowarefoundation/autoware_core_universe_prototype/discussions/367) for the word initialization.
I don't feel so (maybe because I'm a developer).

@isamu-takagi I'm sorry, I couldn't understand what you mean exactly. :pleading_face: 
Could you add a bit more supplemental information? :pray:

Reply:
@kenji-miyake 
Change API and state names https://github.com/autowarefoundation/autoware-documentation/pull/153/commits/73c6cbf8e8fca009a22abc7a0d8208a08c001821.